frontend/.eslintrc

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Should the following `eslintConfig` field in `package.json` be removed now that this file is added?

https://github.com/se701-group6/se701-assignment1/blob/3a306a204e34fd5a53d39aad4d0e7cca6fe940d7/frontend/package.json#L23-L25

`.eslintrc` [takes priority](https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy) over `package.json` so the `package.json` entry might be confusing.